### DISCUSSION TITLE
Is there any 3D model with 2 inputs ?

### DISCUSSION THREAD

**Discussion by gkrisp98** [Project-MONAI/MONAI#4529] [Q&A] [answered]
Hi, I am working on a classification of 3D MRI where I want to combine a mask and a raw MRI and I was wondering if someone has implemented a 3D model with 2 inputs

**Comment by wyli** [comment, marked as answer]
One way to implement it is to make sure the image/mask have the same spatial size and concatenate them at dim=0:
```
monai.transforms.ConcatItemsD(keys=['img1', 'img2'], name='img12', dim=0),
```
for example, this is used in the tutorial: https://github.com/Project-MONAI/tutorials/blob/main/deep_atlas/deep_atlas_tutorial.ipynb
